frontend/app.vue

```diff
@@ -1,6 +1,19 @@
+<script setup>
+const { finalizePendingLocaleChange } = useI18n();
+
+const onBeforeEnter = async () => {
+  await finalizePendingLocaleChange();
+};
+</script>
+
 <template>
-  <div>
-    <NuxtRouteAnnouncer />
-    <NuxtWelcome />
-  </div>
+  <NuxtLayout>
+    <NuxtPage
+      :transition="{
+        name: 'my',
```

Comment:
du hast keine animation mit `my` definiert. die ganze animation ist relativ wertlos wenn wir nicht `skipSettingLocaleOnNavigate: true` in nuxt.config.ts gesetzt haben
ich find nicht mal dass wir unbedingt eine animation brauchen